### DISCUSSION TITLE
[FIXED] Image generation doesnt work with latest 1.63.yr0-ROCm release

### DISCUSSION THREAD

**Discussion by Krxon** [YellowRoseCx/koboldcpp-rocm#41] [General]
Hi

I tried latest release and image generation doesn't work. Nor as standalone or with sillytavern.

It throws this error:
Error Encountered 
Image Generation Failed!
Please make sure A1111 is running and properly configured!
In your local install of Automatic1111 WebUi, modify webui-user.bat and add these flags to enable API access:
set COMMANDLINE_ARGS= --api --listen --cors-allow-origins=*

In previous 1.61.2.yr0-ROCm image generation works OK. Using it on RX 7900 XT

Regards

**Comment by YellowRoseCx** [comment]
What model are you using to try and generate images? I've never tried the new "SD.CPP" image generation that got added, but llama.cpp upstream changed a bunch of stuff and a lot of things got broken :sob: I can try to look into it though

Are you trying to generate images with KoboldCPP or with Automatic1111 and the API?

**Reply by raz334** [reply]
I can confirm that version [KoboldCPP-v1.63.yr1-ROCm]
Does not work to generate images, I had to poke around to find one that did not generate some kind of a "Cuda" error or come back with the same error message the others are getting, which was gotten around by removing the /v1 at the end of the localhost entry in the http statement.
The latest pre-release version as of 5/1/24 at the time of this message simply crashes when trying to load a SD model with the cuda error.

However getting it to load properly without any errors generated and attempting to create an image results it simply saying Generating Image 25 layers.... or something like that. It remains stuck on that message with no indication of progress, and no increase in computer resources being used of any kind. Hope this helps..

**Can you include a progress indicator of some kind so we know when it is actually working?**

**Reply by YellowRoseCx** [reply]
It should now all be working in v1.68 https://github.com/YellowRoseCx/koboldcpp-rocm/releases/tag/v1.68.yr0-ROCm
